examples/gallery/histograms/blockm.py

```diff
@@ -16,13 +16,13 @@
 
 # Set the region for the plot
 region = [130, 152.5, 32.5, 52.5]
-# Define spacing in x and y direction (150 by 150 minute blocks)
+# Define spacing in x and y direction (150x150 minutes blocks)
```

Comment:
```suggestion
# Define spacing in x and y direction (150x150 arc-minutes blocks)
```

Reply:
These changes are made in commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/28d2914d3dcf4a5f378e9517cc98dfb1392fd42e.